### ISSUE TITLE
Documentation of source lists

### ISSUE THREAD

**Issue by jawz101** [badmojr/1Hosts#106]
Many blocklists require that credit is given to them per the licensing of the files.

a) it would be helpful to know what the sources were.  I don't understand mosts lists when they're just lists of lists.  The only combo list I do like is Steven Black's Unified because it's well investigated, well-rounded, and he gives list maintainers a ton of feedback.  I've tested dozens of lists w/ pfSense pfBlocker NG and his has the best coverage, with the least breakages, and least amount of dead hosts (aka bloated, useless lists.)

b) it would be helpful to know what the sources are to report issues upstream

c) you're breaking licenses

Example, @StevenBlack  lists all sources of his blocklists with links to their licenses.  see in his readme.  https://github.com/StevenBlack/hosts  has all of the source lists detailed and links to their licenses.

Your lists are also used in commercial projects such as https://nextdns.io/ which seems iffy
https://github.com/nextdns/metadata/tree/master/privacy/blocklists

@romaincointepas @rs so my complaint is also to NextDNS because you're commercializing other's IP w/o respect of their licenses- many of which simply ask for credit and links to their license

**Comment by badmojr** [comment]
Couple of points I'd like to raise.
- I don't commercially benefit in anyway from maintaining this project.
- A list of "assets" used in making 1Hosts lists are available [here](https://github.com/badmojr/1Hosts/blob/master/statics/assets.txt)
- I have no control on how third-parties choose to use the lists. If any list maintainer likes their list to stop being used, then I would be happy to comply.
- In short, I love to see myself as operating on  good faith. Nothing more, nothing less.
